### PR TITLE
feat: Uploading Clinic App

### PR DESCRIPTION
**Pull request by babitadelfino**

### CODE REVIEW

**Comment by babitadelfino** [comment]
git

**Comment by babitadelfino** [comment]
image

**Comment by babitadelfino** [comment]
this is the landing page

File: apps/clinicapp/clinicapp.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: clinicapp
+description: 'Clinic Management App.'
+website: 'https://github.com/babitadelfino/clinicapp.git'
```
Comment (HashimotoYT) [inline apps/clinicapp/clinicapp.yml]: Your need to replace `website` field with your official website page.

File: apps/clinicapp/clinicapp.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+category: 'Health & Fitness'
+repository: 'https://github.com/babitadelfino/clinicapp.git'
```
Comment (HashimotoYT) [inline apps/clinicapp/clinicapp.yml]: Since this is closed source app you don't need `repository` field.

**Comment by babitadelfino** [comment]
Updated website link, please review

File: apps/clinicapp/clinicapp.yml (hunk 1 above)
Comment (babitadelfino) [reply apps/clinicapp/clinicapp.yml]: Replaced link, please review

**Comment by babitadelfino** [comment]
Please Review

**Comment by babitadelfino** [comment]
Hello Sir,

Please review our changes and let us know if anything blocked from our 
side.

Pull Request Link :- https://github.com/electron/apps/pull/1268

Repo Name is :- babitadelfino/apps

-- 
Regards,

BABITA BHANDARI

**Comment by As-release-etc** [comment]
M_e_i m_e ต้นแบบ​หรือต้นแบบต้น

**Comment by Toinane** [comment]
Hello !
Sorry for the delay.
Has we already said before, you don't have to set the repository field, your git link is a 404 for us.
And I don't see any link to download your app on the website ? How can we download the application?

**Comment by Toinane** [comment]
Hey again, since we didn't received any response, I'll close this PR.
Feel free to reopen it or make a new PR if needed. Thank you!

**Pull request closed without merging**